### PR TITLE
4574: Provenance activity entities should not be indexed

### PR DESCRIPTION
**Pull request by claudiu-cristea**

### CODE REVIEW

File: web/modules/custom/joinup_search/joinup_search.behat.inc [hunk 1]
```diff
@@ -55,23 +55,47 @@ class SearchSubContext extends DrupalSubContextBase implements DrupalSubContextI
...
+   *
+   * @param string $type
+   *   The text of the content tab.
+   *
+   * @throws \Exception
+   *   Thrown when the tab is not found in the page.
+   *
+   * @Given the :type content tab is not displayed
+   */
+  public function assertContentTabNotDisplayed(string $type) {
```
Comment (idimopoulos) [inline web/modules/custom/joinup_search/joinup_search.behat.inc]: I think this step is the same as ::assertContentTypeFacetItemsNotPresent of the same class (the other supports multiple labels) and is not needed. 
The same applies for it's usage on the steps.

Comment (idimopoulos) [reply web/modules/custom/joinup_search/joinup_search.behat.inc]: If you think we can keep it though for a single label check, I am fine with it.

Comment (claudiu-cristea) [reply web/modules/custom/joinup_search/joinup_search.behat.inc]: I was thinking that it's the pair of `assertContentTabDisplayed ()`. Usually we have the "exists" and "not exists" pair. I missed the other one. No strong opinion.

**Comment by idimopoulos** [comment]
I will merge this request myself if green. The last commit is discussed with @claudiu-cristea. No further QA is needed.

**Pull request merged**